docs/sdk/index.mdx

```diff
@@ -87,3 +91,111 @@ Supported SDKs for server-side:
   </div>
 
 </div>
+
+
+## Features available
```

Comment:
I want to have this table right after the client SDK icons.
And for the server, to have the table right after the server SDK icons.

Reply:
Done